src/app/views/App.tsx

```diff
@@ -84,10 +85,27 @@ class App extends Component<IAppProps, IAppState> {
     });
   }
 
+  private collectDeviceCharacteristicsTelemetry() {
+    const deviceProperties = {
+      deviceHeight: screen.height,
+      deviceWidth: screen.width,
+      screenWidth: getDeviceScreenSize(screen.width)
+    };
+
+    telemetry.trackDeviceCharacteristics(deviceProperties);
+  }
+
   public componentDidMount = async () => {
     this.displayToggleButton(this.mediaQueryList);
     this.mediaQueryList.addListener(this.displayToggleButton);
 
+    const hasTelemetryBeenCollected = localStorage.getItem('hasTelemetryBeenCollected');
+
+    if(!hasTelemetryBeenCollected){
```

Comment:
This results in a single piece of telemetry collected per browser, we don't find out what dimensions are being used if the user resizes the screen, which is a very real scenario.

My suggestion is that we capture this information in a way that appears redundant so that we can also look at how many graph requests are being made by screenWidth bucket.